osu.Game/Screens/OnlinePlay/Multiplayer/Participants/ParticipantsList.cs

```diff
@@ -55,6 +59,16 @@ protected override void OnRoomUpdated()
                 // Add panels for all users new to the room.
                 foreach (var user in Room.Users.Except(panels.Select(p => p.User)))
                     panels.Add(new ParticipantPanel(user));
+
+                if (currentHostPanel == null || !currentHostPanel.User.Equals(Room.Host))
+                {
+                    // Reset position of previous host back to normal, if one existing.
+                    if (currentHostPanel != null && panels.Contains(currentHostPanel))
+                        panels.SetLayoutPosition(currentHostPanel, 0);
+
+                    // Change position of new host to display above all participants.
+                    panels.SetLayoutPosition(currentHostPanel = panels.Single(u => u.User.Equals(Room.Host)), -1);
```

Comment:
This `.Single()` call looks dangerous. `Room.Host` is marked nullable, so this may select some random blank panel at best and crash at worst.

Reply:
Good catch, wouldn't have thought `Room.Host` to be nullable, nor did rider note me about that.